website/blog/2020-06-21-radium.md

```diff
@@ -0,0 +1,90 @@
+authorURL: https://twitter.com/tomekgodzik
+authorImageURL: https://avatars1.githubusercontent.com/u/3807253?s=460&v=4
+---
+
+We are excited to announce the release of Metals v0.6.2. The release mostly
+focused on adding bugfixes for the most recent v0.6.1 release, but does include
+several major improvements. This will most likely be the last release before the
+v0.7.0 one.
+
+Metals is a language server for Scala that works with VS Code, Atom, Vim,
+Sublime Text and Emacs. Metals is developed at the
+[Scala Center](https://scala.epfl.ch/) along with contributors from the
+community.
+
+Side note, as of this release we also dropped the `Work in progress` label, since Metals already includes most major features of LSP and behaved pretty stable for past few months.
+
+In this release we merged <TODO> PRs and closed <TODO> issues, full details: <TODO>
+
+## TL;DR
+
+- fallback to `show usages` from `Goto definition` if reference is definition
+  itself
+- support for Scala 2.13
+- dropping all Scala versions but 2.11.12, 2.12.7, 2.12.8 and 2.13.0
+- fixes for importing builds in Gradle, Mill and sbt
+- Metals now works with JDK 11
+- miscellaneous fixes
+
+Check out the website and give Metals a try: https://scalameta.org/metals/
+
+## Fallback to `show usages` from `Goto definition`
+
+Thanks to an awesome contribution from @tanishiking we are now able to provide
+you with information about usages if 'Goto definition' was used on the
+definition itself.
+
+![Fallback](https://i.imgur.com/BT3h0FJ.gif)
+
+## Support for Scala 2.13
+
+A lot of work was done by @gabro to make Metals compatible with Scala 2.13. More
+details about it can be seen in
+[his PR](https://github.com/scalameta/metals/pull/763).
+
+## Dropping older Scala versions
+
+Starting from the next version we are dropping all versions but `2.11.12`, `2.12.7`,
+`2.12.8` and `2.13.0` while deprecating `2.11.12` and `2.12.7`. By deprecating `2.11.12` we want to push users to upgrade if they can as it is hard for Metals to
+support 2.11 with the same levels of features as more recent versions. This come up when making Metals compatible with Scala 2.13 and it turns out it is getting very difficult for us to support multiple Scala versions. It must
+be noted that we're not dropping `2.11.12` anytime soon and even when we will,
+2.11 users will always be able to use an older version of Metals.
+
+# Fixes for auto importing builds
+
+- digest only `.sbt` files at workspace level for sbt projects (thanks
+  @wojciechUrbanski!)
+- fix SemanticDB plugin path on window for Gradle workspaces
+- update default Mill version to 0.4.1 to fix an issue with classpath not
+  containing Scala library
+- Metals now pick up the version from `.mill-version` for `millw` script
+
+## Metals now works with JDK 11
+
+Thanks to the joint effort by @olafurpg and @er1c Metals now works with JDK 11! The effort included fixing several JDK 11 issues and some awesome improvements to the indexing itself. <TODO>
```

Comment:
@olafurpg It's a bit short now - anything you want to add?

Reply:
Done.